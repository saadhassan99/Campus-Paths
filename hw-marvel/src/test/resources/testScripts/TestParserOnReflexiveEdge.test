# test Parser on ReflexiveEdge data to make sure MarvelPath 
# doesn't allow building reflexive edges

# Parse through the data and build the graph
LoadGraph ReflexiveEdge ReflexiveEdge.tsv

# List nodes and edges to see if the graph is built correctly
ListNodes ReflexiveEdge
ListChildren ReflexiveEdge A
ListChildren ReflexiveEdge B
ListChildren ReflexiveEdge C