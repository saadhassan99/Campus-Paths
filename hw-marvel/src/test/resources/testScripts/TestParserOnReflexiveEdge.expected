# test Parser on ReflexiveEdge data to make sure MarvelPath 
# doesn't allow building reflexive edges

# Parse through the data and build the graph
loaded graph ReflexiveEdge

# List nodes and edges to see if the graph is built correctly
ReflexiveEdge contains: A B C
the children of A in ReflexiveEdge are: B(AB)
the children of B in ReflexiveEdge are: A(AB) C(BC)
the children of C in ReflexiveEdge are: B(BC)